### PR TITLE
Add iOS tests support for all screen sizes

### PR DESCRIPTION
**Pull request by friedbunny**
This pull request allows the viewport to fill the larger screens of the iPhone 6/6+.

Support is accomplished by adding `LaunchScreen.xib`:

ios simulator screen shot may 21 2015 2 35 51 pm

/cc @incanus @bleege @1ec5

### CODE REVIEW

**Comment by friedbunny** [comment]
Hrm, going to add blank launch images for older devices.

**Comment by friedbunny** [comment]
Alright, the iOS tests app should now fill the viewport on every device on every iOS version.

**Comment by 1ec5** [comment]
> friedbunny:ios-tests-phablet-support

lol, it’s so big!

File: test/ios/LaunchScreen.xib [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <capability name="Constraints with non-1.0 multipliers" minToolsVersion="5.1"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="iN0-l3-epB">
+            <rect key="frame" x="0.0" y="0.0" width="480" height="480"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <label opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="  Copyright (c) 2015 Mapbox. All rights reserved." textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumFontSize="9" translatesAutoresizingMaskIntoConstraints="NO" id="8ie-xW-0ye">
```
Comment (1ec5) [inline test/ios/LaunchScreen.xib]: Nit: Can I bother you to change Xcode’s default (c) to a Unicode ©?

**Comment by 1ec5** [comment]
Thanks for taking care of this! The black bars were driving me mad.

File: test/ios/LaunchScreen.xib (hunk 1 above)
Comment (friedbunny) [inline test/ios/LaunchScreen.xib]: Hah, of course.

**Pull request merged**